src/modules/common/containers/LanguageSelector.js

```diff
@@ -33,19 +32,19 @@ export class LanguageSelector extends React.Component {
    */
   getLanguageChangeAction (languageCode) {
```

Comment:
This doesn't return an action anymore, right?

Reply:
yes it still does (with exception of categories pathname), but I will adjust this (and change the name) to always return a  path with WEBAPP-231.